app/0.14.x/pdk/index.md

```diff
@@ -20,7 +20,7 @@ The Plugin Development Kit (or "PDK") is set of Lua functions and variables
  `kong.request`, `kong.log`, etc...
 
 
-## Table of Contents
+### Table of Contents
```

Comment:
Please remove the manual Table of Contents and allow the new auto-ToC to work its magic!